vendor/manifest

```diff
@@ -0,0 +1,433 @@
+{
+	"version": 0,
+	"dependencies": [
+		{
+			"importpath": "github.com/armon/consul-api",
```

Comment:
So, I'm not sure if this is coming via some other dependency, but this dependency and some others seem like they aren't used in your code. If they're just things pulled in by recursive dependencies then no big deal, but if they can be removed it's probably worth doing so (I think `gvt` has tools for this, but I'm not completely sure).

Reply:
Ya. I switched between few libs for json parsing and other stuff. It might very well be orphaned. Let me take a look.